packages/features/src/properties.ts

```diff
@@ -0,0 +1,20 @@
+export function isValidProperties(properties: string | string[]): boolean {
+  if (!Array.isArray(properties) && typeof properties !== 'string') {
+    return false;
+  }
+
+  // at least one property should be defined
+  if (properties.length === 0) {
+    return false;
+  }
```

Comment:
I don't know if this is required by the spec. But IMO, it is very common to end up having an empty value or list if the `properties` replies on user input. Throwing error here adds unnecessary panic.

Sending an empty query to the server is not harmful these days. I suggest removing this check.

Reply:
And so does `pygeoapi`:

https://demo.pygeoapi.io/master/collections/dutch_windmills/items?f=json&properties=